docker/Dockerfile

```diff
@@ -0,0 +1,39 @@
+FROM archlinux/base:latest
+LABEL Alad Wenter <alad@archlinux.org>
+
+RUN pacman -Syu --noconfirm base base-devel
```

Comment:
I would add `--needed` flag as well, in case some of the stuff is already part of `archlinux/base`